### ISSUE TITLE
Funcotator's LocatableXsv datasources only annotate the first hit in their data

### ISSUE THREAD

**Issue by jonn-smith** [broadinstitute/gatk#4930]
### Tool(s) or class(es) involved
_Funcotator_
_LocatableXsvFuncotationFactory_

### Description
When multiple entries overlap in LocatableXsv datasources, only the first is added to the output as annotations.

All hits should be added as annotations.

Must determine how to put them into fields so that they are parseable.

**Comment by jonn-smith** [comment]
@LeeTL1220 I don't think this needs to be a beta blocker - the vast majority of the time this will not happen anyway.  

When it does occur, from what I've seen a lot of the data are the same (as in the case of `clinvar_hgmd` - the HGMD IDs are different but the other fields are identical).

**Comment by LeeTL1220** [comment]
Feel free to remove label then.
